src/systemcmds/tests/test_autodeclination.cpp

```diff
@@ -23,7 +23,7 @@ class AutoDeclinationTest : public UnitTest
 
 bool AutoDeclinationTest::autodeclination_check(void)
 {
-	ut_assert("declination differs more than 1 degree", get_mag_declination(47.0, 8.0) - 0.6f < 0.5f);
+	ut_assert("declination differs more than 1 degree", get_mag_declination(47.0, 8.0) - 1.6f < 0.5f);
```

Comment:
Could we just duplicate the boundary condition numbers you've used in the other test?